src/mod/rdp/rdp_params.hpp

```diff
@@ -56,7 +56,7 @@ struct ModRDPParams
     const char * target_user;
     const char * target_password;
     const char * target_host;
-    const char * client_address;
+    char client_address[46];
```

Comment:
So do you suggest rather strncpy() ? It's more safe

Reply:
client_address should remain `char const*`

Reply:
Yes but unfortunately, it can be changed during execution if we want to send proxy address, that's why I need array char :/ (or std::string but I don't have the need to use heap allocation...)